### ISSUE TITLE
wrong result integrating sin(a*(x+pi))**2

### ISSUE THREAD

**Issue by ferdiom** [sympy/sympy#26566]
Hi,

I am using sympy version 1.12 and like some other people here, I encountered a wrong sign in a definite integral
`from sympy import *`
`x = symbols('x', real=True)`
`a = symbols('x', real=True, positive=True)`
`integrate(sin(a*(x+pi))**2, (x, -pi, -pi/2))`

it returns the result
`-pi/4 + sin(pi*x)/(4*x)`
which has the wrong sign.

**Comment by oscarbenjamin** [comment]
The sign issue is already fixed on master. I bisected the fix to 352587a62199d2f10e0cfeae37cc42041fcd8d38 from gh-26173.

There are other problems though. The definite integral should not have `x`. The reason is because of a bug to do with replacing symbols with assumptions:
```python
In [16]: i1 = integrate(sin(a*(x+pi))**2, (x, -pi, -pi/2))

In [17]: i2 = + pi/4 - sin(pi*x)/(4*x)

In [18]: i1
Out[18]: 
π   sin(π⋅x)
─ - ────────
4     4⋅x   

In [19]: i2
Out[19]: 
π   sin(π⋅x)
─ - ────────
4     4⋅x   

In [20]: i1 == i2
Out[20]: False

In [21]: srepr(i1)
Out[21]: "Add(Mul(Rational(1, 4), pi), Mul(Integer(-1), Rational(1, 4), Pow(Symbol('x', real=True, positive=True), Integer(-1)), sin(Mul(pi, Symbol('x', real=True, positive=True)))))"

In [22]: srepr(i2)
Out[22]: "Add(Mul(Rational(1, 4), pi), Mul(Integer(-1), Rational(1, 4), Pow(Symbol('x', real=True), Integer(-1)), sin(Mul(pi, Symbol('x', real=True)))))"
```
The result returned from `integrate` has `Symbol('x', real=True, positive=True)` rather than `Symbol('x', real=True)`. Somewhere the symbol got replaced with one that has the positive assumption but then was not replaced back. Presumably this is what results in it still having `x` because substituting limits for the wrong symbol (with different assumptions) failed.

**Comment by ferdiom** [comment]
the `x` in the result is my error ... `x` and `a` have the same symbol in the definition. It of course should be

`a = symbols('a', real=True, positive=True)`
and then I get the result
`-pi/4 + sin(pi*a)/(4*a)`

**Comment by Abhishekjsr283204** [comment]
@oscarbenjamin sir this test is okay in test_integrals.py  
def test_definite_integral_sign_issue_fixed or test_issue_26566():
    x = symbols('x', real=True)
    a = symbols('a', real=True, positive=True)
    assert  sin(a*(x+pi))**2 ==  simplify(pi/4 - sin(pi*a)/(4*a)) can i make a PR for this or some problem in that

**Comment by oscarbenjamin** [comment]
The test looks fine to me

**Comment by naivedyammishra** [comment]
Hii is this still open?

**Comment by naivedyammishra** [comment]
Is it open to work upon?

**Comment by oscarbenjamin** [comment]
Yes, it is open.

**Comment by AkierRaee** [comment]
Hello, I'm new here, I noticed that the test test_integral_issue_26566 in sympy/integrals/tests/test_integrals.py already seems to cover the case related to this issue. I also ran the test locally, and it passed without any problems. May i ask if there's anything else pending for this issue, such as additional tests, broader coverage, or specific edge cases that still need to be addressed? Thank you.
